### PR TITLE
Bugfix: cancel invite, remove member, search functionality in user ma…

### PR DESCRIPTION
**Pull request by Saranya-jena**
Signed-off-by: Saranya-jena <saranya@chaosnative.com>

## Proposed changes
- fixed bugs from remove and cancel invite in teaming section
- fixed search functionality in user management
- CSS fixes in user management table
Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by SarthakJain26 — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]

File: litmus-portal/frontend/src/views/Settings/TeamingTab/Team/removeMemberModal.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      }
+    >
+      <div className={classes.body}>
+        <img src="./icons/userDel.svg" alt="lock" />
+        <div className={classes.text}>
+          <Typography className={classes.typo} align="center">
+            {isRemove
+              ? t('settings.teamingTab.deleteUser.header')
+              : t('settings.teamingTab.deleteModal.header')}
+            <strong> {row.user_name}?</strong>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Settings/TeamingTab/Team/removeMemberModal.tsx:81]: the classname could have had this fontWeight as bold instead

File: litmus-portal/frontend/src/views/Settings/TeamingTab/Team/removeMemberModal.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          )}
+        </div>
+        <div className={classes.buttonGroup}>
+          <ButtonOutlined
+            onClick={() => {
+              showModal();
+              handleClose();
+            }}
+          >
+            <>{t('settings.teamingTab.deleteUser.noButton')}</>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Settings/TeamingTab/Team/removeMemberModal.tsx:98]: is <> required?

**Review by ishangupta-ds — APPROVED** [review]
@Saranya-jena PTAL

File: litmus-portal/frontend/src/views/Settings/TeamingTab/Team/removeMemberModal.tsx (hunk 1 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/views/Settings/TeamingTab/Team/removeMemberModal.tsx:81]: Is there any specific reason behind not using `<strong>` ?

**Pull request merged**